pymeasure/instruments/pendulum/cnt91.py

```diff
@@ -0,0 +1,182 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2021 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+from time import sleep
+
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import (
+    strict_discrete_set,
+    strict_range,
+    truncated_range,
+)
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class CNT91(Instrument):
+    """Represents a Pendulum CNT-91 frequency counter."""
+
+    CHANNELS = {"A": 1, "B": 2, "C": 3, "E": 4, "INTREF": 6}
+    MAX_BUFFER_SIZE = 32000  # User Manual 8-38
+
+    def __init__(self, resourceName, **kwargs):
+        super().__init__(resourceName, "Pendulum CNT-91", **kwargs)
```

Comment:
```suggestion
        super().__init__(resourceName, "Pendulum CNT-91", timeout = 120000, read_termination = '\n', **kwargs)
```
I think you can apply change above and remove following two lines.

Reply:
Fixed in 3575d75.